static/coffee/directives.coffee

```diff
@@ -30,3 +28,27 @@ directives.directive('cpContact', () ->
     ]      
   }
 )
+
+#----------------------------------------------------------------------------
+# A contact field value
+#----------------------------------------------------------------------------
+directives.directive('cpFieldvalue', () ->
```

Comment:
Thanks, noted. I think the plan for the base pod's UI is to expect all item values to be strings. If we start defining custom pods with values of other types, this will come in quite useful though.